src/views/PurchaseOrder.vue

```diff
@@ -133,11 +133,20 @@ export default defineComponent({
         return !this.fieldMappings[id] ? id : this.generateUniqueMappingPrefId();
       },
       saveMapping() {
-        if (this.mappingName) {
+        if (this.mappingName && this.file && this.areAllFieldsSelected()) {
+          showToast(translate("Mapping saved successfully"));
```

Comment:
It should after the operation completion

Reply:
yes sir. Fixed it